src/plugins/dashboard/public/application/utils/get_nav_actions.tsx

```diff
@@ -381,11 +376,6 @@ export const getNavActions = (
           'data-test-subj': 'saveDashboardSuccess',
         });
 
-        const appPath = `${createDashboardEditUrl(id)}`;
-
-        // Manually insert a new url so the back button will open the saved visualization.
-        history.replace(appPath);
-        // setActiveUrl(appPath);
```

Comment:
Is this behavior moved elsewhere? This is a vital piece of user experience.

Reply:
Yea this is added under the TODO list that is part of URL navigations.